### PR TITLE
Add legacy SHC revocation RFC

### PR DESCRIPTION
**Pull request by christianpaquin**
RFC for legacy card revocation, as discussed in #207.

TODO:
 - Depends on PR #205. Update text after merge.
 - Add some of the commentary from discussion #207.
 - Give better guidance on method selection.

### CODE REVIEW

**Review by jmandel — APPROVED** [review]
A couple of notes that needn't prevent merging!

File: rfcs/002-legacy-shc-revocation.md
Comment (christianpaquin) [reply rfcs/002-legacy-shc-revocation.md:19]: That's a good point. I like the idea of adding canonicalization (using [RFC8785](https://datatracker.ietf.org/doc/html/rfc8785) rules?)

Comment (christianpaquin) [reply rfcs/002-legacy-shc-revocation.md]: Added canonicalization to the proposal.

**Comment by christianpaquin** [comment]
I'm considering removing the `hmac-patient` method from this rfc; doesn't seem to meet the requirements of the issuers we talked to. Anyone planning to use it?

**Comment by christianpaquin** [comment]
FYI @jmandel, since you approved a month ago; I made some updates to link to the revocation artefacts (now merged) in the spec, and I removed the hmac-patient method that no one expressed interest in.

**Review by jmandel — APPROVED** [review]

**Comment by jmandel** [comment]
Thanks! Still looks good, and indeed even better. Now:-) If you have permission to merge you can go ahead. Otherwise I am happy to press the button. Thanks! Still looks good, and indeed even better now :)

I'd say you can go ahead and merge; We can have a pretty low threshold for RFC content at draft status.

**Comment by christianpaquin** [comment]
> I'd say you can go ahead and merge; We can have a pretty low threshold for RFC content at draft status.

I don't have write access to the repo, so I'll let you merge @jmandel.

**Pull request merged**
